recipes/LJSpeech/quantization/README.md

```diff
@@ -1,74 +1,102 @@
+--data_folder data/LJSpeech \
+--ssl_hub microsoft/wavlm-large \
+--n_clusters 1000 \
+--layer_id 7 \
+--experiment_name wavlm_K1000_L7
+```
 
+---------------------------------------------------------------------------------------------------------
 
+## Results
+
+The output folders with checkpoints and logs can be found [here](https://www.dropbox.com/sh/bk5qz0u1ppx15jk/AAAj23FI3AVKtfRKGvyHJYHza?dl=0).
+
+The checkpoints can be also found at [this](https://huggingface.co/speechbrain/SSL_Quantization) HuggingFace repository.
+
+**NOTE**: these logs and checkpoints were created using an earlier version of the code.
+
+---------------------------------------------------------------------------------------------------------
+
+## About SpeechBrain
 
-# **About SpeechBrain**
 - Website: https://speechbrain.github.io/
 - Code: https://github.com/speechbrain/speechbrain/
 - HuggingFace: https://huggingface.co/speechbrain/
 
+---------------------------------------------------------------------------------------------------------
+
+## Citing SpeechBrain
 
-# **Citing SpeechBrain**
 Please, cite SpeechBrain if you use it for your research or business.
 
 ```bibtex
-@misc{speechbrainV1,
-  title={Open-Source Conversational AI with SpeechBrain 1.0},
-  author={Mirco Ravanelli and Titouan Parcollet and Adel Moumen and Sylvain de Langen and Cem Subakan and Peter Plantinga and Yingzhi Wang and Pooneh Mousavi and Luca Della Libera and Artem Ploujnikov and Francesco Paissan and Davide Borra and Salah Zaiem and Zeyu Zhao and Shucong Zhang and Georgios Karakasidis and Sung-Lin Yeh and Pierre Champion and Aku Rouhe and Rudolf Braun and Florian Mai and Juan Zuluaga-Gomez and Seyed Mahed Mousavi and Andreas Nautsch and Xuechen Liu and Sangeet Sagar and Jarod Duret and Salima Mdhaffar and Gaelle Laperriere and Mickael Rouvier and Renato De Mori and Yannick Esteve},
-  year={2024},
-  eprint={2407.00463},
-  archivePrefix={arXiv},
-  primaryClass={cs.LG},
-  url={https://arxiv.org/abs/2407.00463},
+@article{ravanelli2024open,
+  author  = {Mirco Ravanelli and Titouan Parcollet and Adel Moumen and Sylvain de Langen and Cem Subakan and Peter Plantinga and Yingzhi Wang and Pooneh Mousavi and Luca {Della Libera} and Artem Ploujnikov and Francesco Paissan and Davide Borra and Salah Zaiem and Zeyu Zhao and Shucong Zhang and Georgios Karakasidis and Sung-Lin Yeh and Pierre Champion and Aku Rouhe and Rudolf Braun and Florian Mai and Juan Zuluaga-Gomez and Seyed Mahed Mousavi and Andreas Nautsch and Ha Nguyen and Xuechen Liu and Sangeet Sagar and Jarod Duret and Salima Mdhaffar and Ga{{\"e}}lle Laperri{{\`e}}re and Mickael Rouvier and Renato De Mori and Yannick Est{{\`e}}ve},
+  title   = {Open-Source Conversational {AI} with {SpeechBrain} 1.0},
+  journal = {Journal of Machine Learning Research},
+  year    = {2024},
+  volume  = {25},
+  number  = {333},
+  pages   = {1--11},
+  url     = {http://jmlr.org/papers/v25/24-0991.html}
 }
```

Comment:
I understnad that this is the official new citation, but you could please replace `ravanelli2024open` by `speechbrainV1` to remain consistent with our past formatting. I will do the modifications for the other ones.